packages/block-library/src/columns/style.scss

```diff
@@ -10,41 +10,37 @@
 }
 
 .wp-block-column {
-	flex: 1;
+	flex-grow: 1;
 	margin-bottom: 1em;
 
 	// Responsiveness: Show at most one columns on mobile.
 	flex-basis: 100%;
 
-	// Beyond mobile, allow 2 columns.
-	@include break-small() {
-		flex-basis: 50%;
-		flex-grow: 0;
```

Comment:
`flex-grow:0;` is the default, so if we remove this above, we no longer need it here.